WordPress/gutenberg
# Created author block.

## Description
This PR aims to add an author block, as specified in issue https://github.com/WordPress/gutenberg/issues/1844..

## Screenshots
<img width="895" alt="screen shot 2017-11-01 at 13 32 12" src="https://user-images.githubusercontent.com/11271197/32279997-4c1b1cf8-bf12-11e7-836e-bf8da3136be3.png">
<img width="545" alt="screen shot 2017-11-01 at 13 32 01" src="https://user-images.githubusercontent.com/11271197/32279981-39890168-bf12-11e7-96ad-76057ecb7357.png">
<img width="912" alt="screen shot 2017-11-01 at 13 32 24" src="https://user-images.githubusercontent.com/11271197/32280006-5583e108-bf12-11e7-8652-c1467bd55a10.png">
<img width="585" alt="screen shot 2017-11-01 at 13 32 42" src="https://user-images.githubusercontent.com/11271197/32280016-60a68af4-bf12-11e7-90d5-d18b8fde2b8a.png">
<img width="898" alt="screen shot 2017-11-01 at 13 32 51" src="https://user-images.githubusercontent.com/11271197/32280024-66210f4a-bf12-11e7-874e-5222215ee982.png">
<img width="582" alt="screen shot 2017-11-01 at 13 33 18" src="https://user-images.githubusercontent.com/11271197/32280027-693a14a6-bf12-11e7-9fbc-2172066b4a6c.png">
<img width="891" alt="screen shot 2017-11-01 at 13 33 30" src="https://user-images.githubusercontent.com/11271197/32280033-6d06994c-bf12-11e7-8ae7-c39df975957d.png">
<img width="884" alt="screen shot 2017-11-01 at 13 33 38" src="https://user-images.githubusercontent.com/11271197/32280043-717cdb9e-bf12-11e7-9071-1c7f8b013474.png">

## To test
Add an author block, try to hide each of three options see the hiding attribute works as expected in the editor and when saving the post.
Try each of the alignment options verify the alignment looks ok.

## Notes
The single-author component is going to be reused in author blocks.
A max-width of 50% was added when align is not center, because without the post after published does not look as in the editor and in fact, no difference is noticeable when compared to center alignment.
